ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -228,12 +228,12 @@ Examples:
 
 /* eslint-enable */
 
-function ShortNamify(name) {
+function ShortNamify(name, options) {
```

Comment:
Maybe just to pass in `PlayerNicks` is better in this usage.

Reply:
Done, thanks!